setup/docker.md

```diff
@@ -17,7 +17,21 @@ Coder for Docker works with the following platforms:
   [Docker Desktop 20.10](https://www.docker.com/products/docker-desktop).
   **Note**: Coder for Docker requires Windows Subsystem for Linux at this time.
 
-Coder requires x86-64 and does not support ARM-based processors at this time.
+### For systems without x86-64 architecture
```

Comment:
@jawnsy what do you think of the header? Not sure how to phrase this -- I wanted `non-x86-64 architecture systems` but that had too many hyphens :)

Reply:
I guess "Non-x86_64 systems" might work, avoids the hyphen problem

Trivia: The actual name for this architecture is amd64, and Intel licenses it from AMD, so many systems refer to it with that name. Intel had their own ia64 (Itanium) architecture but it's defunct.

https://en.wikipedia.org/wiki/X86-64